docs/rules/no-import-module-exports.md

```diff
@@ -75,4 +77,5 @@ module.exports = foo;
 ```
 
 ### Further Reading
- - [webpack issue #4039](https://github.com/webpack/webpack/issues/4039)
+
+- [webpack issue #4039](https://github.com/webpack/webpack/issues/4039)
```

Comment:
why wouldn't the list be indented by a space?

Reply:
The first list level is not indented. I've never indented it and seems unnecessary to do so.

https://github.com/DavidAnson/markdownlint/blob/v0.29.0/doc/md007.md

Reply:
what would the diff look like if we set `start_indented` to `true` and `start_indent` to 1? that's my usual style

Reply:
It looks like it will match your desired style then. I pushed a commit for this.